notes-service/src/main/java/org/exoplatform/wiki/service/impl/NoteServiceImpl.java

```diff
@@ -2085,11 +2113,6 @@ private Map<String, String> copyNotePageProperties(Page oldNote,
       MetadataItem newNoteMetadataItem = getNoteMetadataItem(note, lang, newObjectType);
       if (oldNoteMetadataItem != null && oldNoteMetadataItem.getProperties() != null) {
         properties = oldNoteMetadataItem.getProperties();
-        if (properties != null && oldLang == null && lang != null) {
-          properties.remove(FEATURED_IMAGE_ID);
-          properties.remove(FEATURED_IMAGE_ALT_TEXT);
-          properties.remove(FEATURED_IMAGE_UPDATED_DATE);
-        }
         if (newNoteMetadataItem != null) {
           newNoteMetadataItem.setProperties(properties);
```

Comment:
no more need to properties initialization IMO, you can use directly newNoteMetadataItem.setProperties(oldNoteMetadataItem.getProperties())